buildtest/cli/config.py

```diff
@@ -183,7 +187,20 @@ def list_profiles(configuration, theme=None, print_yaml=None):
     for profile_name in configuration.target_config["profiles"].keys():
         print(profile_name)
 
+def remove_profiles(configuration, theme=None, print_yaml=None):
```

Comment:
please remove `theme` and `print_yaml` method they are not required

Reply:
done